src/main/java/org/cristalise/storage/jooqdb/JooqClusterStorage.java

```diff
@@ -82,6 +83,7 @@ public void initialiseHandlers() throws PersistencyException {
         jooqHandlers.put(ClusterType.COLLECTION, new JooqCollectionHadler());
         jooqHandlers.put(ClusterType.HISTORY,    new JooqHistoryHandler());
         jooqHandlers.put(ClusterType.JOB,        new JooqJobHandler());
+        jooqHandlers.put(ClusterType.ATTACHMENT,        new JooqOutcomeAttachmentHandler());
```

Comment:
Please keep the alignments of these declarations. It is done for code readability.